### DISCUSSION TITLE
Jittering video received

### DISCUSSION THREAD

**Discussion by percof** [bluenviron/mediamtx#1168] [Questions]
I don't if it is just me, but I'm having problem of using the rtsp server "SMOOTHLY". My use case is quite simple: I want to loop-play a local mp4 file to simulate a camera, which typically has a rtsp server for streaming. I start rtsp-simple-server without changing any configuration, and use ffmpeg to playback the mp4 file:
ffmpeg -re -stream_loop -1 -i test.mp4 -c copy -f rtsp rtsp://localhost:8554/mystream.
And then, I use vlc to pull the stream from the server on the same computer:
vlc rtsp://localhost:8554/mystream
No doubt, I'm able to play the video in vlc. But the problem is the video received in vlc is very jittering and lagging. I played with ffmpeg by trying different configs (like bufsize, transcode etc.). The severity of jittering varies a bit, but never goes away.  How do I fix this? Am I missing anything here?  Thanks.

**Comment by aler9** [comment]
Hello, try one of the techniques described here:
https://github.com/aler9/rtsp-simple-server#corrupted-frames

[a comment by an automated account is not shown]
